ground/gcs/src/plugins/telemetryscheduler/telemetryschedulergadgetwidget.cpp

```diff
@@ -1089,7 +1089,6 @@ void QFrozenTableViewWithCopyPaste::init()
                                    "color: #FF0000;"
                                    "background-color: #8EDE21;"
                                    "selection-background-color: #999}"); //for demo purposes
-    frozenTableView->setSelectionModel(selectionModel());

```

Comment:
"setSelectionModel() failed: Trying to set a selection model, which works on a different model than the view". This happens because it is trying to take the selection model from the parent object and set the child object (frozenTableView) but the parent object doesn't share the model ([set on the child object right above](https://github.com/tracernz/TauLabs/blob/6c0975773631ef8aa7299c2b59b377fd8ad171b3/ground/gcs/src/plugins/telemetryscheduler/telemetryschedulergadgetwidget.cpp#L1080)). I'm fairly certain this doesn't make sense and it was failing anyhow?